cloud/yarn.lock

```diff
@@ -0,0 +1,1633 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
oh yarn. when we port into the rewrite branch we gotta remember the change to npm